src/util.tsx

```diff
@@ -143,22 +148,25 @@ export function unquote_cmdline(text) {
     return words;
 }
 
-export function image_name(image) {
+export function image_name(image: { RepoTags?: string[] }): string {
```

Comment:
Hmm, a proper `Image` type would be nice, no? Even if it starts out with just `RepoTags` in it for now.

Reply:
It would indeed be nice, but that requires some more code shifting. utils.tsx is at the bottom of the module stack, and the `Image` type belongs more into rest.tsx or client.tsx. So either we need to move the function, or move the all the podman API types out into yet another file (perhaps that's best?). I added a comment, that's something for the next batch when I don't have to drag around a node_modules refresh (that goes stale in 5 minutes)